test/sanity/code-smell/docs-build.requirements.txt

```diff
@@ -3,5 +3,6 @@ pyyaml
 resolvelib
 sphinx
 sphinx-notfound-page
+sphinx_ansible_theme
```

Comment:
I think either this should be restricted to a specific version of the theme, or https://github.com/ansible-community/sphinx_ansible_theme/pull/32 should get merged and this PR adjusted before this PR should be merged, because that PR will break the docsite if this PR is kept as-is.

Reply:
That PR won't be merged for the time being so it's unnecessary + this specific requirements file is for linters. There's another one with a precise pin.